resources/dicts/patrols/plains/border/leaf-fall.json

```diff
@@ -39,5 +39,3633 @@
+        },
+        {
+            "text": "p_l bolts back to camp, {PRONOUN/p_l/poss} heart pounding. {PRONOUN/p_l/subject/CAP} have to tell {PRONOUN/p_l/poss} clanmates. {PRONOUN/p_l/subject/CAP} burst into camp yowling a warning!",
+            "exp": 40,
+            "weight": 10,
+            "art": "running_warrior",
+            "relationships": [
+                {
+                    "cats_to": ["p_l"],
+                    "cats_from": ["clan"],
+                    "mutual": false,
+                    "values": ["trust", "respect"],
+                    "amount": 15
+                }
+            ]
+        },
+        {
+            "text": "Frozen in the face of the large threat, {PRONOUN/s_c/subject} musters a surge of courage. As the coyote comes into sight {PRONOUN/s_c/subject/} {VERB/s_c/manage/manages} to flee from the coyote, sprinting back to camp to warn the clan.",
+            "exp": 40,
+            "weight": 20,
+            "art": "running_warrior",
+            "stat_trait": ["insecure", "nervous"],
+            "relationships": [
+                {
+                    "cats_to": ["s_c"],
+                    "cats_from": ["clan"],
+                    "mutual": false,
+                    "values": ["trust", "respect"],
+                    "amount": 10
+                }
+            ]
+        },
+        {
+            "text": "s_c doesn't have much time to think as {PRONOUN/s_c/subject} {VERB/s_c/hear/hears} the yips. The coyote has {PRONOUN/s_c/poss} scent. Seeing some Thunderbeast dung nearby, s_c dives for it, rolling around in it before slinking through the tall grass, being careful to keep hidden. It's a tense moment, but s_c {VERB/s_c/manage/manages} to make it home safe.",
+            "exp": 40,
+            "weight": 20,
+            "art": "gen_hunt_strange",
+            "stat_skill": ["HUNTER,2", "CLEVER,2", "INSIGHTFUL,2"]
+        },
+        {
+            "text": "s_c hears the yips. The coyote has {PRONOUN/s_c/poss} scent. {PRONOUN/s_c/subject/CAP} run. Runs as fast as {PRONOUN/s_c/subject} can because {PRONOUN/s_c/poss} life depends on it. But s_c can't run forever, the thudding footsteps are catching up.  {PRONOUN/s_c/subject/CAP} see a group of sleeping Thunderbeasts, and dart to them, yowling. As a Thunderbeast slowly stands, the coyote turns tail and flees.",
+            "exp": 40,
+            "weight": 20,
+            "art": "gen_coyote1",
+            "stat_skill": ["RUNNER,2", "CLEVER,2", "INSIGHTFUL,2"]
+        },
+        {
+            "text": "s_c hears the yips. The coyote has {PRONOUN/s_c/poss} scent.  {PRONOUN/s_c/subject/CAP} run. Run as fast as {PRONOUN/s_c/subject} can because {PRONOUN/s_c/poss} life depends on it. But s_c can't run forever, the thudding footsteps are catching up.  {PRONOUN/s_c/subject/CAP} see a group of sleeping Thunderbeasts, and dart to them, yowling. s_c watches with a new respect and fear as the Thunderbeasts kill the predator with ease.",
```

Comment:
```suggestion
            "text": "s_c hears the yips. The coyote has {PRONOUN/s_c/poss} scent.  {PRONOUN/s_c/subject/CAP} {VERB/s_c/run/runs}. {VERB/s_c/run/runs/CAP} as fast as {PRONOUN/s_c/subject} can because {PRONOUN/s_c/poss} life depends on it. But s_c can't run forever, the thudding footsteps are catching up.  {PRONOUN/s_c/subject/CAP} {VERB/s_c/see/sees} a group of sleeping Thunderbeasts, and {VERB/s_c/dart/darts} to them, yowling. s_c watches with a new respect and fear as the Thunderbeasts kill the predator with ease.",
```